examples/create-job-by-nodename.js

```diff
@@ -0,0 +1,35 @@
+
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+
+export default function () {
+  const kubernetes = new Kubernetes({
+    // config_path: "/path/to/kube/config", ~/.kube/config by default
+  })
+
+  const nodes = kubernetes.nodes.list();
+  const nodeName = nodes[0].namespace;
```

Comment:
Guessing you actually meant `nodes[0].name` and _not_ namespace.

Reply:
should be fixed by https://github.com/grafana/xk6-kubernetes/pull/29/commits/5a66a5f42d1da5bc0d9e64cdc58a232200a4e91f